### ISSUE TITLE
Task lists NOT rendering correctly...

### ISSUE THREAD

**Issue by mralexgray** [ShyykoSerhiy/gfm-plugin#95]
Here is my markdown...

```
- [x] Easy.
- [x] Single / multiple item-select.
- [x] For commandline nodejs terminal apps.
- [x] Optional cancel message.
```

here is my rendered preview...

![screen shot 2015-10-14 at 12 50 00 am](https://cloud.githubusercontent.com/assets/262517/10475109/85a4ba82-720d-11e5-9c34-eae9e33e63ca.png)

[Not right...] (https://guides.github.com/features/mastering-markdown/#)

**Comment by jguice** [comment]
Hi, same problem here.

gfm: 0.1.8
IntelliJ Ultimate: 14.1.5

**Comment by ShyykoSerhiy** [comment]
Yeah, it looks like I've messed up with both offline AND online generation for task list. I'll integrate support for https://github.com/github/task_list in 0.1.9.

**Comment by jguice** [comment]
Ah, cool, thanks!
